### PR TITLE
Choreo

### LINKED ISSUE #30: Choreo

### PR DESCRIPTION
**Pull request by Lyssia-Seiden**
we p much need this merged today

### CODE REVIEW

**Comment by Awesomeplayer165** [comment]
Looking into it now

File: src/main/java/frc/lib/choreolib/AutoFieldPosition.java
Comment (Awesomeplayer165) [inline src/main/java/frc/lib/choreolib/AutoFieldPosition.java:1]: write on same line?

File: src/main/java/frc/lib/choreolib/ChoreoPath.java
Comment (Awesomeplayer165) [inline src/main/java/frc/lib/choreolib/ChoreoPath.java:1]: Remove unused code

**Comment by Awesomeplayer165** [comment]
Are we going to include 3 piece bump?

File: src/main/java/frc/robot/subsystems/Grabber/GrabberSubsystem.java
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/Grabber/GrabberSubsystem.java:1]: Not necessary now, but do we need to have every method with Command? Should't that be inferred from the return type?

My guess is that you are trying to switch all subsystems to using commands so we do don't need to wrap void method calls in an `InstantCommand`.

Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/Grabber/GrabberSubsystem.java:1]: Not necessary now, but do we need to have every method with Command? Should't that be inferred from the return type?

My guess is that you are trying to switch all subsystems to using commands so we do don't need to wrap void method calls in an InstantCommand.

**Comment by Awesomeplayer165** [comment]
lgtm

**Review by Awesomeplayer165 — APPROVED** [review]

**Comment by Lyssia-Seiden** [comment]
> Are we going to include 3 piece bump?

im going to skip rn since idt we have time to test, we can run the 2 piece if needed or pathplanner autos

**Pull request merged**